medicai/transforms/random/random_elastic_transform.py

```
@@ -0,0 +1,286 @@
+
+def _smooth_along_axis(
+    x: tf.Tensor,
+    kernel_1d: tf.Tensor,
+    axis: int,
+    radius: int,
+    spatial_rank: int,
+) -> tf.Tensor:
+    """Apply one separable smoothing pass along one spatial axis."""
+    kernel_shape = [1] * spatial_rank
+    kernel_shape[axis] = int(kernel_1d.shape[0])
+    kernel = tf.reshape(kernel_1d, kernel_shape + [1, 1])
```

Ruff (0.15.21)

[warning] Consider `[*kernel_shape, 1, 1]` instead of concatenation

Replace with `[*kernel_shape, 1, 1]`

(RUF005)